Use DM to Migrate Sharded Tables (within 1 TB).md

```diff
@@ -0,0 +1,395 @@
+---
+title: Use DM to Migrate Sharded Tables (within 1 TB)
+summary: Introduces how to use DM to migrate sharded tables (within 1 TB).
+---
+
+# Consolidate MySQL Shards into TiDB (up to 1 TiB)
+
+If you want to merge and migrate multiple MySQL database instances upstream to one TiDB database downstream, and the amount of data is not too large (for example, the sum of all sharded tables is less than 1 TB), you can use DM to migrate sharded tables. Through examples in this article, you can learn the operation steps, precautions, and troubleshooting of the migration.
+
+This document applies to migration of sharded tables within 1 TB in total.
+
+If you want to migrate sharded tables with a total of more than 1 TB of data, it will take a long time by using DM. It is recommended that you follow the operation introduced in [Using Dumpling and TiDB Lightning to migrate sharded table data](/migrate-from-mysql-shard-merge-using-lightning.md) to migrate data larger than 1 TB.
```

Comment:
```suggestion
If you want to migrate sharded tables with a total of more than 1 TiB of data, it will take a long time by using DM. It is recommended that you follow the operation introduced in [Using Dumpling and TiDB Lightning to migrate sharded table data](/migrate-from-mysql-shard-merge-using-lightning.md) to migrate data larger than 1 TiB.
```